.github/workflows/code-scanning.yml

```
@@ -0,0 +1,84 @@
+name: Code scanning

```

```suggestion
name: Code scanning by Fortify

```
Include information about the tool, to distinguish from other providers.